pkg/cluster/internal/create/actions/loadbalancer/haproxy.go

```diff
@@ -60,6 +60,16 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 	ctx.Status.Start("Starting the external load balancer ⚖️")
 	defer ctx.Status.End(false)
 
+	err = ConfigHAProxy(loadBalancerNode, allNodes)
+	if err != nil {
+		return err
+	}
+
+	ctx.Status.End(true)
+	return nil
+}
+
+func ConfigHAProxy(loadBalancerNode *nodes.Node, allNodes []nodes.Node) error {
```

Comment:
linter is failing in this line with the following error
> /home/prow/go/src/sigs.k8s.io/kind/pkg/cluster/internal/create/actions/loadbalancer/haproxy.go:72:1: exported function ConfigHAProxy should have comment or be unexported

Reply:
Thanks. :+1:  Addressed.